### PR TITLE
Artist page formatting

### PR DESCRIPTION
**Pull request by localastronaut**
Couldn't figure out how to get the formatting to work for the lists. It sees they link back to something on the backend

### CODE REVIEW

File: templates/artist/artist_list.html [hunk 1]
```diff
@@ -28,10 +28,10 @@
                 </ul>
             </li>
         </ul>
-        music
-        <button class="button hollow" id="location-dropdown-button" type="button" data-toggle="location-dropdown">
+        <h5 class="sort-options">music</h5>
+        <button class="button hollow location" id="location-dropdown-button" type="button" data-toggle="location-dropdown">
             {% if distance %}within {{ distance }} miles{% endif %}
-            from {% if lat and lon %}my location{% elif location %}{{ location }}{% else %}Anywhere{% endif %}
+            <h5 class="sort-options">FROM</h5> {% if lat and lon %}my location{% elif location %}{{ location }}{% else %}<h5 class="sort-options">ANYWHERE</h5>{% endif %}
```
Comment (RevolutionTech) [inline templates/artist/artist_list.html]: Seems like here you would want `my location` to be all caps as well.

File: templates/registration/profile.html [hunk 2]
```diff
@@ -7,15 +7,15 @@
 
 {% block content %}
     <h1>Profile</h1>
-    <h2 class="profile-header">Good morning {% firstof user.get_full_name user.get_username %}!</h2>
```
Comment (RevolutionTech) [inline templates/registration/profile.html]: These changes are in a different branch. You should revert them here.

File: templates/registration/public_profile.html [hunk 3]
```diff
@@ -2,33 +2,31 @@
 {% load humanize %}
```
Comment (RevolutionTech) [inline templates/registration/public_profile.html]: These changes are in a different branch, you should revert them here.

Comment (RevolutionTech) [inline templates/registration/public_profile.html]: Actually nvm, I just noticed this branch was built on top of that one. This is fine.

File: templates/registration/profile.html (hunk 2 above)
Comment (RevolutionTech) [inline templates/registration/profile.html]: Actually nvm, I just noticed this branch was built on top of that one. This is fine.

**Comment by RevolutionTech** [comment]
Not sure what you meant for "Couldn't figure out how to get the formatting to work for the lists.". You mean the different genres and sorting options come from the backend?

**Comment by localastronaut** [comment]
On the list the "genre" and "filters" I cant change the padding to get it to align with the rest of the sentence. Also, can't get the words to be uppercase or apply any styling to the lists

File: templates/artist/artist_list.html [hunk 4]
```diff
@@ -28,10 +28,10 @@
...
         </button>
         <div class="dropdown-pane" id="location-dropdown" data-dropdown>
             within
```
Comment (RevolutionTech) [inline templates/artist/artist_list.html]: Seems like you would also want to put `within` in all caps also.

**Comment by RevolutionTech** [comment]
Alright, I gave you a hand here. Applied styling in the way I think you wanted. Mostly, anyways. We probably want the dropdowns to have slightly smaller text though, I'm thinking.

**Comment by localastronaut** [comment]
Want to get the two dropdown menus to line up horizontally with the rest of the sentence so it flows together. Still gets bumped up 

Also, agree with the text being too big. I'll change it

screen shot 2016-05-19 at 1 24 43 am

**Comment by RevolutionTech** [comment]
Yeah, I agree. I haven't attempted to line up the sentence. I expect it will take a bit of playing around.

**Pull request merged**